docs/mp/openapi/01_openapi.adoc

```diff
@@ -178,19 +156,224 @@ MP OpenAPI supports a number of other mandated settings. These are described in
 link:{mp-openapi-spec}#configuration[configuration section] of the MicroProfile 
 OpenAPI spec.
 
-include::{common-openapi-doc}[common-config]
-Assign these settings in `META-INF/microprofile-config.properties`.
 
+== REST Endpoint
 
-== Accessing the OpenAPI document
-Now your Helidon MP application will automatically respond to an additional endpoint --
- `/openapi` -- and it will return the OpenAPI document describing the endpoints
+The Helidon MP application will automatically respond to the built-in endpoint --
+`/openapi` -- and it will return the OpenAPI document describing the endpoints
 in your application.
 
-By default, per the MicroProfile OpenAPI spec, the default format of the OpenAPI document is YAML. 
+By default, per the MicroProfile OpenAPI spec, the default format of the OpenAPI document is YAML.
 There is not yet an adopted IANA YAML media type, but a proposed one specifically
 for OpenAPI documents that has some support is `application/vnd.oai.openapi`.
 That is what Helidon returns, by default.
 
-In addition a client can specify the HTTP header `Accept:` as either `application/vnd.oai.openapi+json` or `application/json`
+In addition, a client can specify the HTTP header `Accept:` as either `application/vnd.oai.openapi+json` or `application/json`
 to request JSON. Alternatively, the client can pass the query parameter `format` as either `JSON` or `YAML` to receive `application/json` or `application/vnd.oai.openapi` (YAML) output, respectively.
+
+== Configuration
+
+Helidon OpenAPI configuration is supported by the following class:
```

Comment:
I was not clear in my earlier comments about this.

In MP, developers are very unlikely to use the config _class_. They will, of course, use the config _settings_.

Since the links need to be corrected so you'll be in the file one more time, maybe change this sentence to something like:

Helidon OpenAPI configuration supports the following settings: